### PR TITLE
[FP-535] Add bigquery client library

### PR DESCRIPTION
**Pull request by 15joeybloom**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // The following APIs and libraries are available at runtime:
+// * @google-cloud/bigquery@6.0.1 as google.cloud.bigquery
```
Comment (pevans) [inline index.js:2]: Is the leading `@` symbol intentional?

**Review by pevans — COMMENTED** [review]
Just one question

File: index.js (hunk 1 above)
Comment (15joeybloom) [reply index.js:2]: Yep! That's the name of the package, for some reason. https://www.npmjs.com/package/@google-cloud/bigquery

Comment (pevans) [reply index.js:2]: Huh 🤷

**Review by pevans — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (15joeybloom) [reply index.js:2]: https://docs.npmjs.com/about-scopes

It's a scoping thing; npm gives google exclusive access to the `@google-cloud` namespace. Anything not starting with an @ is open season.

**Pull request merged**